kafka-rest/src/main/java/io/confluent/kafkarest/resources/v3/BrokerConfigsResource.java

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright 2018 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.kafkarest.resources.v3;
+
+import io.confluent.kafkarest.Versions;
+import io.confluent.kafkarest.controllers.BrokerConfigManager;
+import io.confluent.kafkarest.entities.BrokerConfig;
+import io.confluent.kafkarest.entities.v3.BrokerConfigData;
+import io.confluent.kafkarest.entities.v3.BrokerData;
+import io.confluent.kafkarest.entities.v3.ClusterData;
+import io.confluent.kafkarest.entities.v3.CollectionLink;
+import io.confluent.kafkarest.entities.v3.GetBrokerConfigResponse;
+import io.confluent.kafkarest.entities.v3.ListBrokerConfigsResponse;
+import io.confluent.kafkarest.entities.v3.ResourceLink;
+import io.confluent.kafkarest.response.CrnFactory;
+import io.confluent.kafkarest.response.UrlFactory;
+import java.util.Comparator;
+import java.util.Objects;
+import java.util.concurrent.CompletableFuture;
+import java.util.stream.Collectors;
+import javax.ws.rs.GET;
+import javax.ws.rs.NotFoundException;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.container.AsyncResponse;
+import javax.ws.rs.container.Suspended;
+
+@Path("/v3/clusters/{clusterId}/brokers/{brokerId}/configs")
+public class BrokerConfigsResource {
+
+  private final BrokerConfigManager brokerConfigManager;
+  private final CrnFactory crnFactory;
+  private final UrlFactory urlFactory;
+
+
+  public BrokerConfigsResource(
+      BrokerConfigManager brokerConfigManager,
+      CrnFactory crnFactory,
+      UrlFactory urlFactory) {
+    this.brokerConfigManager = Objects.requireNonNull(brokerConfigManager);
+    this.crnFactory = Objects.requireNonNull(crnFactory);
+    this.urlFactory = Objects.requireNonNull(urlFactory);
+  }
+
+  @GET
+  @Produces(Versions.JSON_API)
+  public void listBrokerConfigs(
+      @Suspended AsyncResponse asyncResponse,
+      @PathParam("clusterId") String clusterId,
+      @PathParam("brokerId") String brokerId) {
+    CompletableFuture<ListBrokerConfigsResponse> response =
+        brokerConfigManager.listBrokerConfigs(clusterId, brokerId)
+          .thenApply(
+               configs ->
+                   new ListBrokerConfigsResponse(
+                       new CollectionLink(
+                           urlFactory.create(
+                               "v3", "clusters", clusterId, "brokers", brokerId, "configs"),
+                           null),
```

Comment:
Add a comment like `/* next= */  null`, otherwise it's pretty hard to tell what this argument is.